### ISSUE TITLE
Fails on Windows Deluge 2.0.3: No module named 'twisted.internet._dumbwin32proc'

### ISSUE THREAD

**Issue by Sahasrahla** [d8ahazard/deluge-extractor#21]
Here was my build process:

- Hyper-V VM with Windows Server 2019 Core (no gui)
- Install Deluge 2.0.3 from [here](https://forum.deluge-torrent.org/viewtopic.php?f=12&t=55776&p=232351#p23235), specifically deluge-2.0.3-lt1.2.10-boost1.74-py3.8.5-ossl1.1.1g-GTK3.24.23.exe
- Install 7-zip 64-bit, specifically 7z1900-x64.exe
- Installed SimpleExtractor from this repo via Deluge WebUI, specifically SimpleExtractor-1.4-py3.8.egg
- Started Deluge-debug with verbose logging
- Downloaded a torrent containing a .rar set

What I got was a torrent that downloaded just fine, Deluge moved it from "\\fileserver\downloading" to "\\fileserver\seeding" as it should and marked the torrent Finished, and the Extractor started.

`23:48:25 [INFO    ][deluge_simpleextractor.core   :69  ] Extracting \\fileserver\sonarr-seeding\specific-torrent-folder\specific-torrent-name.rar from [some long ID string] with C:\Program Files\7-Zip\7z.exe x -y to \\fileserver\sonarr-seeding\specific-torrent-folder`
`23:48:25 [ERROR   ][deluge.core.eventmanager      :79  ] Event handler TorrentFinishedEvent failed in <bound method Core._on_torrent_finished of <deluge_simpleextractor.core.Core object at 0x000001EA17954700>> with exception No module named 'twisted.internet._dumbwin32proc'
`

**Comment by Sahasrahla** [comment]
Alright, I've gone down the rabbit hole a little bit on this.

tl;dr the plugin is fine, and the issue was with the Unofficial Deluge 2 Windows Installer. I upgraded to deluge-2.0.4.dev56.exe (found in the same Deluge Forum thread I linked in the top post) and now SimpleExtractor works as intended.

For future reference, I found that I was able to get past the "No module named 'twisted.internet._dumbwin32proc'" error by [copying _dumbwin32proc.py from the Twisted github repo](https://github.com/twisted/twisted/blob/trunk/src/twisted/internet/_dumbwin32proc.py) and putting it in %DelugeInstallDirectory%\Lib\site-packages\twisted\internet. After that was resolved, I was still getting an error when using this plugin, now it was about a module named win32security, which I found was part of PyWin32. I was not able to repeat the trick because I couldn't find where the PyWin32 files were located in the Deluge directory, but as stated above, simply upgrading to one of the Dev releases of Deluge 2.0.4 brought all the needed dependencies along and your plugin is working great for me now.
